e2e/helpers/entityViewPageFixtures.ts

```diff
@@ -7,4 +7,10 @@ export const contents = `<h1>My entity view</h1>
     <Value path="label" />
   </li>
 </Repeat>
-<br />`;
+<br />
+
+<p id='dataset-content'></p>
+`;
+
+export const script = `var datasets = window.store.getState().page.datasets.getIn(['currentEntity', 'sharedId']);
```

Comment:
This variable is called incorrectly.  This variable is not `datasets`, it is `currentEntitySharedId`. Still, I was thinking it could be useful to `append` this to the actual script, so that users could have access directly to, precisely: `datasets` inside the script, instead of having to getState().

What do you think @daneryl ?

Reply:
Regardless of the matter to discuss, I renamed the variable as suggested.

Reply:
@RafaPolit what do you mean ? append the datasets by default ? this is precisely what will cause size issues. if someone is advanced enough to write a custom script i think they can manage to work with the store no ?

Reply:
@daneryl I mean include in the page script, forced, in the first line:
```
var datasets = window.store.getState().page.datasets.toJS();
```
That way, users have access to the `datasets` variable without needing to know our store's internal structure, our store's type of objects (page is a JS object, but datasets is an Immutable).

This, more or less, the same as the previous approach in terms of user experience, yet it does not actually include all the datasets in the HTML on server side rendering,  It's just a one-liner string.  It would task the browser with more memory and extra processing, yes.  It's finding the balance between useful and optimized.

What do you think?